projectTemplates/ts-server/dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:12.18.3 as build
+WORKDIR /usr/src/app
+COPY ./package*.json ./
+RUN npm install
+COPY . .
+RUN npm run build
+#reomve node modules and install only production dependencies
+#this step is require beacuse npm install fails on alpine
+RUN rm -r ./node_modules && npm install --only=production
+
+
+FROM node:12.18.3-alpine3.11 as production
+ARG NODE_ENV=production
+ENV NODE_ENV=${NODE_ENV}
+WORKDIR /usr/app
+#TODO: uncomment for confd
+#COPY ./run.sh ./run.sh
+#RUN chmod +x run.sh
+COPY package*.json ./
+COPY ./api ./api
+#TODO: replace for confd
+#COPY ./confd ./confd
+COPY ./config ./config
+COPY --from=build /usr/src/app/dist .
+COPY --from=build /usr/src/app/node_modules ./node_modules
+#TODO: replace for confd
+#CMD ["./run.sh"]
+CMD ["node", "./index.js"]
```

Comment:
Setup a health check for the container [Dockerfile HEALTHCHECK](https://docs.docker.com/engine/reference/builder/#healthcheck)